src/main/java/com/pokegoapi/api/PokemonGo.java

```diff
@@ -26,34 +26,19 @@
 import com.pokegoapi.main.RequestHandler;
 import com.pokegoapi.util.SystemTimeImpl;
 import com.pokegoapi.util.Time;
-import lombok.Getter;
-import lombok.Setter;
```

Comment:
@FabianTerhorst my IDE probably auto-removed it because I HAD made all the get/set synchronized. As I look at them now it seems that I lose them in one of my changes and need to add  synchronized back in.